### ISSUE TITLE
Logits all reduced to very small value when training multi-label image classification

### ISSUE THREAD

**Issue by Lancerchiang** [tensorflow/tensorflow#19890]
```
Tensorflow version: 1.7-gpu
Operating system: ubuntu 16.04
```

I was training a multi-label image classification model (an image could have multiple labels, and the label is like `[1,0,0,0,1,0,0,1]` with variable number of 1s).

I have switched to the `sigmoid_cross_entropy_with_logits` like below:

```python
with slim.arg_scope(inception_resnet_v2_arg_scope()):
            logits, end_points = inception_resnet_v2(images, num_classes=dataset.num_classes, is_training=True)
         
        exclude = ['InceptionResnetV2/Logits', 'InceptionResnetV2/AuxLogits']
        variables_to_restore = slim.get_variables_to_restore(exclude=exclude)

        fp_labels = tf.cast(labels, tf.float32)
        loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=fp_labels, logits=logits)
        
        total_loss = tf.losses.get_total_loss()  
        
        global_step = get_or_create_global_step()

        # Define your exponentially decaying learning rate
        lr = tf.train.exponential_decay(
            learning_rate=initial_learning_rate,
            global_step=global_step,
            decay_steps=decay_steps,
            decay_rate=learning_rate_decay_factor,
            staircase=True)

        # Now we can define the optimizer that takes on the learning rate
        optimizer = tf.train.AdamOptimizer(learning_rate=lr)

        # Create the train_op.
        train_op = slim.learning.create_train_op(total_loss, optimizer)
        
        def train_step(sess, train_op, global_step):
   
            # Check the time for each sess run
            start_time = time.time()
            total_loss, global_step_count, _ = sess.run([train_op, global_step, metrics_op])
            time_elapsed = time.time() - start_time

            # Run the logging to print some results
            logging.info('global step %s: loss: %.4f (%.2f sec/step)', global_step_count, total_loss, 
            time_elapsed)

            return total_loss, global_step_count

```

But when I started training the loss is decreasing very fast (from 0.5 to almost 0 in several minites), and all the logits value became almost zero for every image. 

I tried manual loss function implementation but also failed.

Any advice would be greatly appreciated!

**Comment by Lancerchiang** [comment]
I resolved it by replacing `tf.nn.sigmoid_cross_entropy_with_logits` by `tf.losses.sigmoid_cross_entropy`, though I don't know the reason...

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
